k8s/memcached/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,55 @@
 actions:
-- name: dummy test
+- name: StatefulSet nodes deployed successfully
```

Comment:
You can use example of NGINX instead. It seems to be simpler.

https://github.com/GoogleCloudPlatform/click-to-deploy/blob/50a06a4ca4c803ab39c00e91f75166c62c13c9fc/k8s/nginx/apptest/tester/tests/basic-suite.yaml.template#L2-L17

Reply:
It will be even easier in the future, because of this change: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/309

Reply:
That's great suggestion, thanks!